crates/sui-core/src/authority/authority_store_tables.rs

```diff
@@ -566,10 +564,16 @@ fn effects_table_default_config() -> DBOptions {
 }
 
 fn events_table_default_config() -> DBOptions {
-    default_db_options()
-        .optimize_for_write_throughput()
-        .optimize_for_large_values_no_scan(4 << 10)
-        .optimize_for_read(read_size_from_env(ENV_VAR_EVENTS_BLOCK_CACHE_SIZE).unwrap_or(1024))
+    DBOptions {
+        options: default_db_options()
+            .optimize_for_write_throughput()
+            .optimize_for_large_values_no_scan(4 << 10)
+            .optimize_for_read(read_size_from_env(ENV_VAR_EVENTS_BLOCK_CACHE_SIZE).unwrap_or(1024))
+            .options,
+        rw_options: ReadWriteOptions {
+            apply_range_deletions: true,
```

Comment:
yes but events table has existing call site that uses range delete. Didn't want to change existing behavior in case if it relies on deletion to be observed instantly